bots/create-and-run-a-bot/rasa-bot/README.md

```diff
@@ -11,95 +11,93 @@ You will need Python knowledge only when creating custom actions in Rasa which c
 The fastest way to start with Rasa is using [Rasa starter kit](https://github.com/RocketChat/rasa-kick-starter).
 The starter kit uses webhooks to communicate between Rasa and Rocket.Chat.
 
-1. **clone the starter repo**
+**1. clone the starter repo**
 
-    ```
-    git clone https://github.com/RocketChat/rasa-kick-starter
-    cd rasa-kick-starter/
-    ```
+```
+git clone https://github.com/RocketChat/rasa-kick-starter
+cd rasa-kick-starter/
+```
 
-2. **configure the bot**
+**2. configure the bot**
 
-    1. open `rasa-kick-starter/bot_rasa` folder and update the `credentials.yml` file with Rasa bot's username, password, and Rocket.Chat URL:
+* open `rasa-kick-starter/bot_rasa` folder and update the `credentials.yml` file with Rasa bot's username, password, and Rocket.Chat URL:
 
-        ```yaml
-        rocketchat:
-          user: "<RASA USER NAME>"
-          password: "<RASA USER PASS>"
-          server_url: "<ROCKETCHAT HOST>"
-        ```
+```yaml
+rocketchat:
+user: "<RASA USER NAME>"
+password: "<RASA USER PASS>"
+server_url: "<ROCKETCHAT HOST>"
+```
 
-    1. train the bot's Machine Learning Model:
+* train the bot's Machine Learning Model:
 
-        ```
-        docker run -it -v $(pwd)/bot_rasa:/app rasa/rasa train
-        ```
+```
+docker run -it -v $(pwd)/bot_rasa:/app rasa/rasa train
+```
 
-        After the training, the machine learning model will be created inside the `bot_rasa/models` folder.        
+After the training, the machine learning model will be created inside the `bot_rasa/models` folder.
 
-3. **run the bot:**
+**3. run the bot**
 
-    ```
-    docker-compose up bot_rasa
-    ```
-  
-    You should see the following output:
+```
+docker-compose up bot_rasa
+```
 
-    ```
-    $ docker-compose up bot_rasa
-    Starting rasa-kick-starter_bot_rasa_1 ... done
-    Attaching to rasa-kick-starter_bot_rasa_1
-    bot_rasa_1            | 2019-10-31 21:23:24 INFO     root  - Starting Rasa server on http://localhost:5005
-    ```
+You should see the following output:
 
-    Open your browser and navigate to the `http://localhost:5005`. You should see the response from the running Rasa bot:
+```
+$ docker-compose up bot_rasa
+Starting rasa-kick-starter_bot_rasa_1 ... done
+Attaching to rasa-kick-starter_bot_rasa_1
+bot_rasa_1            | 2019-10-31 21:23:24 INFO     root  - Starting Rasa server on http://localhost:5005
+```
 
-    ```
-    Hello from Rasa: 1.5.0a1
-    ```
+Open your browser and navigate to the `http://localhost:5005`. You should see the response from the running Rasa bot:
```

Comment:
no 'the' required

navigate to `http://localhost:5005`

Reply:
Fixed